### DISCUSSION TITLE
How to set presentation: "modal" for nested navigators?

### DISCUSSION THREAD

**Discussion by sebastienlabine** [react-navigation/react-navigation#9804] [Q&A]
How to use presentation: "modal" for nested navigators? I can always set it for the parent navigator, but I don't want all the screens in the navigator to be a modal, only a few. When I set it up for the child navigator, nothing works.

**Comment by satya164** [comment]
Modals should always be at the root for them to work correctly. For nesting, it's better to avoid nesting stacks, use a single stack with both modal and non-modal screens: https://reactnavigation.org/docs/modal/#creating-a-stack-with-modal-screens
